extensions/amp-sidebar/1.0/sidebar.js

```diff
@@ -230,8 +254,11 @@ function SidebarWithRef(
       if (backdropAnimation) {
         backdropAnimation.cancel();
       }
+      if (documentElement) {
```

Comment:
This also points to separate effect - it'd be much cleaner to cleanup.

Reply:
Updated in the latest!